src/views/StandaloneDirectorsFiling.vue

```diff
@@ -491,7 +482,7 @@ export default class StandaloneDirectorsFiling extends Vue {
   }
 
   /** Called just before this component is destroyed. */
-  protected beforeDestroy (): void {
```

Comment:
I think you didn't need to remove these visibility modifiers (since these methods are not called from the template, which is the problem we were trying to solve).

Anyway, no big deal because the modifiers do nothing at the moment except make developer a bit harder. No change needed, but now you know.

Reply:
Resolving.